### PR TITLE
MDBOOK: Harvest FAQ from proto into MDBOOK -329

### PR DESCRIPTION
**Pull request by murrayjbrown**
@Connoropolous

### CODE REVIEW

**Comment by Connoropolous** [comment]
Ah, oh wow @murrayjbrown ... this is intense. Is this content from throughout the wiki?

**Comment by murrayjbrown** [comment]
@Connoropolous It's the FAQ page from the wiki. From the way the issues are worded, I presumed that you wanted the FAQ as a separate MDBOOK, whereas for #330 the rest of the wiki content is to be migrated into the _holochain_101_ MDBOOK. Is that right?

**Comment by Connoropolous** [comment]
I wasn't thinking strictly of doing it separately, no, I hadn't considered it actually. I think it could be integrated? What do you think?

**Comment by murrayjbrown** [comment]
It's certainly useful to include the FAQ within the _holochain_101_ MDBOOK. There may or may not be utility in packaging a separate FAQ MDBOOK. So, was the separation of #329 and #330 more a staging matter? I can do it either way.

**Comment by murrayjbrown** [comment]
D'oh, looking at it again, now, I see that I'd neglected to remove the unreferenced wiki files.

The landing page is _Home.md_ and the FAQ is _FAQ.md_ and they share some included files but most are referenced by _Home.md._ Where did you intend the non-FAQ wiki content to be published? I'd thought parts of it was to be folded into the _holochain_101_ MDBOOK, as and where appropriate.

**Comment by Connoropolous** [comment]
To your last point, precisely. In one sense, it would be best to be pulled out and into different areas where appropriate. That part could wait for a little further thought to go into it.

**Comment by murrayjbrown** [comment]
Okay, good. So let me go ahead and move the FAQ into the _holochain_101_ MDBOOK for issue #329.

**Comment by Connoropolous** [comment]
@murrayjbrown so is this good for review again?

**Comment by murrayjbrown** [comment]
Yes. It will need some content revisions, though, as noted by TODOs in the document, when updated information is available.

File: doc/holochain_101/src/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+2. There's no mining on Holochain. Blockchain's proof-of-work system provides a hefty incentive for thousands of people to spend the processing power of their CPUs and GPUs using up [huge amounts](https://digiconomist.net/bitcoin-energy-consumption) [of electricity](https://motherboard.vice.com/en_us/article/ywbbpm/bitcoin-mining-electricity-consumption-ethereum-energy-climate-change) on solving a meaningless cryptographic puzzle. Holochain doesn't have mining.
+
+# How is Holochain different from __________?
+
+**TODO: Update with reference to Rust project.**
+
+Please see the [Comparisons page](https://github.com/Holochain/holochain-proto/wiki/Comparisons).
+
+# Who is on the development team? What is their history and experience?
```
Comment (Connoropolous) [inline doc/holochain_101/src/FAQ.md]: Let's take this section out from here entirely. The right place for that info will be somewhere else.

Comment (murrayjbrown) [reply doc/holochain_101/src/FAQ.md]: @Connoropolous Sure. Just the Comparisons section?

Comment (Connoropolous) [reply doc/holochain_101/src/FAQ.md]: Nah, the "who is on the dev team / history /experience"

Comment (murrayjbrown) [reply doc/holochain_101/src/FAQ.md]: @Connoropolous Okay, will do.

Comment (murrayjbrown) [reply doc/holochain_101/src/FAQ.md]: @Connoropolous BTW, I can add navigation next/previous page links to the MDBOOK via a simple build option, if you like.

Comment (zippy) [reply doc/holochain_101/src/FAQ.md]: agreed re removing team.  Out of date and wrong place...  That was more relevant back in ICO days.

**Comment by murrayjbrown** [comment]
@zippy Will do.

**Comment by murrayjbrown** [comment]
@Connoropolous @zippy Requested changes ready for review. I merged latest commits, resolving a flagged conflict, fixed some links, FAQ header levels, and added that MDBOOK build option for previous/next page links.

File: doc/holochain_101/book.toml [hunk 2]
```diff
@@ -3,3 +3,6 @@ authors = ["David Meister"]
 multilingual = false
 src = "src"
 title = "holochain_101"
+
+[build]
+preprocess = ["links"]
```
Comment (Connoropolous) [inline doc/holochain_101/book.toml]: what does this do?

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ This is the home of the Holochain Rust library, being rewritten from [Go](https:
 **[Code Status:](https://github.com/holochain/holochain-rust/milestones?direction=asc&sort=completeness&state=all)** Rust version is currently Pre-Alpha. Not for production use. The code has not yet undergone a security audit. We expect to destructively restructure code APIs and data chains until Beta. Prototype go version was unveiled at our first hackathon (March 2017), with go version Alpha 0 was released October 2017.  Alpha 1 was released May 2018.  We expect a developer pre-release of this Rust re-write in mid October 2018.
 <br/>
 
-| Holochain Links: | [FAQ](https://github.com/holochain/holochain-proto/wiki/FAQ) | [Developer Docs](https://holochain.github.io/holochain-rust/) | [White Paper](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf) |
+| Holochain Links: | [FAQ](https://github.com/holochain/holochain-rust/wiki/FAQ) | [Developer Docs](https://holochain.github.io/holochain-rust/) | [White Paper](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf) |
```
Comment (Connoropolous) [inline README.md]: won't it be: https://holochain.github.io/holochain-rust/faq.html

File: doc/holochain_101/book.toml (hunk 2 above)
Comment (murrayjbrown) [reply doc/holochain_101/book.toml]: @Connoropolous It adds the prev/next page links to the MDBOOK.

File: README.md (hunk 3 above)
Comment (murrayjbrown) [reply README.md]: @Connoropolous  Sorry, yeah, I just realized myself that a simple substitution wasn't right. Your link is close but the _FAQ.html_ base name is uppercase (as was the original); I'll change that to lowercase, if you prefer. Probably better that way.

Comment (murrayjbrown) [reply README.md]: @Connoropolous Okay, I've pushed that change, with the FAQ in a renamed _faq.md_ file and links updated accordingly.

File: doc/holochain_101/book.toml (hunk 2 above)
Comment (Connoropolous) [reply doc/holochain_101/book.toml]: Cool, I like it

**Pull request merged**